docs/src/developing_zed__building_zed.md

```diff
@@ -11,7 +11,9 @@ git submodule update --init --recursive
 ## Dependencies
 
 - Install [Rust](https://www.rust-lang.org/tools/install)
-- Install [Xcode](https://apps.apple.com/us/app/xcode/id497799835?mt=12) from the macOS App Store
+- Install [Xcode](https://apps.apple.com/us/app/xcode/id497799835?mt=12) from the macOS App Store, or from the [Apple Developer](https://developer.apple.com/download/all/) website. Note this requires a developer account, which is free.
```

Comment:
```suggestion
- Install [Xcode](https://apps.apple.com/us/app/xcode/id497799835?mt=12) from the macOS App Store, or from the [Apple Developer](https://developer.apple.com/download/all/) website. Note this requires a developer account.
```

`free` is a loaded term, let's not use it.

Reply:
Great point, thanks! Also thanks for doing that edit.

I was trying to think of a better term to use, as I didn't want readers to think that it was tied to the $99USD/year Apple Developer Program fee.